src/components/CarouselDots.js

```diff
@@ -17,14 +17,14 @@ class CarouselDots extends Component {
     const numberOfSlides = this.props.number || this.props.thumbnails.length;
     const moduloItem = this.calculateButtonValue() % numberOfSlides;
 
-    return this.props.onChange(this.props.value - ( moduloItem - index));
+    return this.props.onChange(this.props.value - (moduloItem - index));
```

Comment:
could you configure lint to fail in such a situation?

Reply:
We may move this comment to a separate ticket. I did it as a The Boy Scout Rule.

Reply:
OK so you can create a ticket and paste its link here

Reply:
https://github.com/adam-golab/eslint-config-brainhub/pull/23